app/controllers/purchases_controller.rb

```diff
@@ -5,6 +5,10 @@ def index
     @purchases = current_user.purchases.all
   end
 
+  def show
```

Comment:
Where is the show controller used? For single view of the purchase? Like recipt?